meetings/2021-02-22.md

```diff
@@ -0,0 +1,85 @@
+# 2021-02-22 Authentication Panel
+
+## Meeting
+ * Call: https://meet.jit.si/solid-authentication
+
+## Present
+
+* C Sarven
+* Henry Story
+* Elf Pavlik
+* TomJ
+* Aaron Coburn
+
+## Agenda
+
+* [Adding meeting notes from 2021-01-25](https://github.com/solid/authentication-panel/pull/122/files) -- (2 minutes)
+*  [SolidCG+WICG review/preparation](https://github.com/WICG/WebID/issues/54#issuecomment-779405599) (20 minutes)
+*  [PR #125: Http Sig extension for Credentials and DIDs](https://github.com/solid/authentication-panel/pull/125): 
+    *  updates? 
+    *  give time for discussion wrap-up / merge by 2021-03-01? Next steps? (15 minutes)
+* [Issue 128: OIDC - Document implementations](https://github.com/solid/authentication-panel/issues/128) (3 minutes?)
+* Solid-OIDC next steps (15 minutes)
+* Default branch name (2 minutes)
+
+## Notes
+
+### Meeting Notes
+
+PR of previous notes accepted. 
+
+### SolidCG+WICG 
+
+* [Slides for TPAC 2020 talk](https://lists.w3.org/Archives/Public/www-archive/2020Oct/att-0010/The_Web_Platform__Privacy_and_Federation_-_TPAC.pdf) on WICG WebID
+* [WebID TPAC 2020 Video](https://www.youtube.com/watch?v=PMfp2qPW-RY)
+
+We will present WebID, OIDC and learn what their ideas are.
+One hour after this call.
+
+Meeting will be on [Feb 22 at 17:00 GMT](https://meet.jit.si/solid-wicg-identity-authentication)
+
+Questions:
+* What's the definition of WebID? (Henry)
+  * Name collision - next steps (Sarven)
+* State of token binding (Aaron)
+* Mechanisms in the browser to differentiate between ClientIDs/Relying Party (Aaron)
+* bblfish: I would be interested to know if they have looked at [Universal Wallet](https://w3c-ccg.github.io/universal-wallet-interop-spec/) 
+
+### PR 125: HttpSig
+
+[Http Sig Original branch](https://github.com/bblfish/authentication-panel/blob/HttpSig/HttpSignature.md)
+
+Questions: ..
+- Elf: who can access and use private key? Can an app have a private key?
+  + bblfish: Yes, and App can create its private key and sign messages with it using JS Crypto. 
+    Indeed the use delegated credentails to give rights to the App as needed.
+- Q proof of possesion - is it not the same as PoP ? 
+- Henry: it is probably very similar. I get lost in the complexity of OIDC.
+  It might be a reason to specify such a minimal version on which more complex protocols such as OIDC can built, but also on which other protocols using SSI stack can build
```

Comment:
```suggestion
  It might be a reason to specify such a minimal version on which more complex protocols such as OIDC can be built, but also on which other protocols using SSI stack can build
```